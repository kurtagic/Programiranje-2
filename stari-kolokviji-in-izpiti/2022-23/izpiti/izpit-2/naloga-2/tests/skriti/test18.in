pkcNCpRlCm 1 10
