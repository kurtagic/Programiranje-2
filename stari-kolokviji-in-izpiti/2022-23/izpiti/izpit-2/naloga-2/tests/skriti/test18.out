p|k|c|N|C|p|R|l|C|m
p|k|c|N|C|p|R|l|Cm
p|k|c|N|C|p|R|lC|m
p|k|c|N|C|p|R|lCm
p|k|c|N|C|p|Rl|C|m
p|k|c|N|C|p|Rl|Cm
p|k|c|N|C|p|RlC|m
p|k|c|N|C|p|RlCm
p|k|c|N|C|pR|l|C|m
p|k|c|N|C|pR|l|Cm
p|k|c|N|C|pR|lC|m
p|k|c|N|C|pR|lCm
p|k|c|N|C|pRl|C|m
p|k|c|N|C|pRl|Cm
p|k|c|N|C|pRlC|m
p|k|c|N|C|pRlCm
p|k|c|N|Cp|R|l|C|m
p|k|c|N|Cp|R|l|Cm
p|k|c|N|Cp|R|lC|m
p|k|c|N|Cp|R|lCm
p|k|c|N|Cp|Rl|C|m
p|k|c|N|Cp|Rl|Cm
p|k|c|N|Cp|RlC|m
p|k|c|N|Cp|RlCm
p|k|c|N|CpR|l|C|m
p|k|c|N|CpR|l|Cm
p|k|c|N|CpR|lC|m
p|k|c|N|CpR|lCm
p|k|c|N|CpRl|C|m
p|k|c|N|CpRl|Cm
p|k|c|N|CpRlC|m
p|k|c|N|CpRlCm
p|k|c|NC|p|R|l|C|m
p|k|c|NC|p|R|l|Cm
p|k|c|NC|p|R|lC|m
p|k|c|NC|p|R|lCm
p|k|c|NC|p|Rl|C|m
p|k|c|NC|p|Rl|Cm
p|k|c|NC|p|RlC|m
p|k|c|NC|p|RlCm
p|k|c|NC|pR|l|C|m
p|k|c|NC|pR|l|Cm
p|k|c|NC|pR|lC|m
p|k|c|NC|pR|lCm
p|k|c|NC|pRl|C|m
p|k|c|NC|pRl|Cm
p|k|c|NC|pRlC|m
p|k|c|NC|pRlCm
p|k|c|NCp|R|l|C|m
p|k|c|NCp|R|l|Cm
p|k|c|NCp|R|lC|m
p|k|c|NCp|R|lCm
p|k|c|NCp|Rl|C|m
p|k|c|NCp|Rl|Cm
p|k|c|NCp|RlC|m
p|k|c|NCp|RlCm
p|k|c|NCpR|l|C|m
p|k|c|NCpR|l|Cm
p|k|c|NCpR|lC|m
p|k|c|NCpR|lCm
p|k|c|NCpRl|C|m
p|k|c|NCpRl|Cm
p|k|c|NCpRlC|m
p|k|c|NCpRlCm
p|k|cN|C|p|R|l|C|m
p|k|cN|C|p|R|l|Cm
p|k|cN|C|p|R|lC|m
p|k|cN|C|p|R|lCm
p|k|cN|C|p|Rl|C|m
p|k|cN|C|p|Rl|Cm
p|k|cN|C|p|RlC|m
p|k|cN|C|p|RlCm
p|k|cN|C|pR|l|C|m
p|k|cN|C|pR|l|Cm
p|k|cN|C|pR|lC|m
p|k|cN|C|pR|lCm
p|k|cN|C|pRl|C|m
p|k|cN|C|pRl|Cm
p|k|cN|C|pRlC|m
p|k|cN|C|pRlCm
p|k|cN|Cp|R|l|C|m
p|k|cN|Cp|R|l|Cm
p|k|cN|Cp|R|lC|m
p|k|cN|Cp|R|lCm
p|k|cN|Cp|Rl|C|m
p|k|cN|Cp|Rl|Cm
p|k|cN|Cp|RlC|m
p|k|cN|Cp|RlCm
p|k|cN|CpR|l|C|m
p|k|cN|CpR|l|Cm
p|k|cN|CpR|lC|m
p|k|cN|CpR|lCm
p|k|cN|CpRl|C|m
p|k|cN|CpRl|Cm
p|k|cN|CpRlC|m
p|k|cN|CpRlCm
p|k|cNC|p|R|l|C|m
p|k|cNC|p|R|l|Cm
p|k|cNC|p|R|lC|m
p|k|cNC|p|R|lCm
p|k|cNC|p|Rl|C|m
p|k|cNC|p|Rl|Cm
p|k|cNC|p|RlC|m
p|k|cNC|p|RlCm
p|k|cNC|pR|l|C|m
p|k|cNC|pR|l|Cm
p|k|cNC|pR|lC|m
p|k|cNC|pR|lCm
p|k|cNC|pRl|C|m
p|k|cNC|pRl|Cm
p|k|cNC|pRlC|m
p|k|cNC|pRlCm
p|k|cNCp|R|l|C|m
p|k|cNCp|R|l|Cm
p|k|cNCp|R|lC|m
p|k|cNCp|R|lCm
p|k|cNCp|Rl|C|m
p|k|cNCp|Rl|Cm
p|k|cNCp|RlC|m
p|k|cNCp|RlCm
p|k|cNCpR|l|C|m
p|k|cNCpR|l|Cm
p|k|cNCpR|lC|m
p|k|cNCpR|lCm
p|k|cNCpRl|C|m
p|k|cNCpRl|Cm
p|k|cNCpRlC|m
p|k|cNCpRlCm
p|kc|N|C|p|R|l|C|m
p|kc|N|C|p|R|l|Cm
p|kc|N|C|p|R|lC|m
p|kc|N|C|p|R|lCm
p|kc|N|C|p|Rl|C|m
p|kc|N|C|p|Rl|Cm
p|kc|N|C|p|RlC|m
p|kc|N|C|p|RlCm
p|kc|N|C|pR|l|C|m
p|kc|N|C|pR|l|Cm
p|kc|N|C|pR|lC|m
p|kc|N|C|pR|lCm
p|kc|N|C|pRl|C|m
p|kc|N|C|pRl|Cm
p|kc|N|C|pRlC|m
p|kc|N|C|pRlCm
p|kc|N|Cp|R|l|C|m
p|kc|N|Cp|R|l|Cm
p|kc|N|Cp|R|lC|m
p|kc|N|Cp|R|lCm
p|kc|N|Cp|Rl|C|m
p|kc|N|Cp|Rl|Cm
p|kc|N|Cp|RlC|m
p|kc|N|Cp|RlCm
p|kc|N|CpR|l|C|m
p|kc|N|CpR|l|Cm
p|kc|N|CpR|lC|m
p|kc|N|CpR|lCm
p|kc|N|CpRl|C|m
p|kc|N|CpRl|Cm
p|kc|N|CpRlC|m
p|kc|N|CpRlCm
p|kc|NC|p|R|l|C|m
p|kc|NC|p|R|l|Cm
p|kc|NC|p|R|lC|m
p|kc|NC|p|R|lCm
p|kc|NC|p|Rl|C|m
p|kc|NC|p|Rl|Cm
p|kc|NC|p|RlC|m
p|kc|NC|p|RlCm
p|kc|NC|pR|l|C|m
p|kc|NC|pR|l|Cm
p|kc|NC|pR|lC|m
p|kc|NC|pR|lCm
p|kc|NC|pRl|C|m
p|kc|NC|pRl|Cm
p|kc|NC|pRlC|m
p|kc|NC|pRlCm
p|kc|NCp|R|l|C|m
p|kc|NCp|R|l|Cm
p|kc|NCp|R|lC|m
p|kc|NCp|R|lCm
p|kc|NCp|Rl|C|m
p|kc|NCp|Rl|Cm
p|kc|NCp|RlC|m
p|kc|NCp|RlCm
p|kc|NCpR|l|C|m
p|kc|NCpR|l|Cm
p|kc|NCpR|lC|m
p|kc|NCpR|lCm
p|kc|NCpRl|C|m
p|kc|NCpRl|Cm
p|kc|NCpRlC|m
p|kc|NCpRlCm
p|kcN|C|p|R|l|C|m
p|kcN|C|p|R|l|Cm
p|kcN|C|p|R|lC|m
p|kcN|C|p|R|lCm
p|kcN|C|p|Rl|C|m
p|kcN|C|p|Rl|Cm
p|kcN|C|p|RlC|m
p|kcN|C|p|RlCm
p|kcN|C|pR|l|C|m
p|kcN|C|pR|l|Cm
p|kcN|C|pR|lC|m
p|kcN|C|pR|lCm
p|kcN|C|pRl|C|m
p|kcN|C|pRl|Cm
p|kcN|C|pRlC|m
p|kcN|C|pRlCm
p|kcN|Cp|R|l|C|m
p|kcN|Cp|R|l|Cm
p|kcN|Cp|R|lC|m
p|kcN|Cp|R|lCm
p|kcN|Cp|Rl|C|m
p|kcN|Cp|Rl|Cm
p|kcN|Cp|RlC|m
p|kcN|Cp|RlCm
p|kcN|CpR|l|C|m
p|kcN|CpR|l|Cm
p|kcN|CpR|lC|m
p|kcN|CpR|lCm
p|kcN|CpRl|C|m
p|kcN|CpRl|Cm
p|kcN|CpRlC|m
p|kcN|CpRlCm
p|kcNC|p|R|l|C|m
p|kcNC|p|R|l|Cm
p|kcNC|p|R|lC|m
p|kcNC|p|R|lCm
p|kcNC|p|Rl|C|m
p|kcNC|p|Rl|Cm
p|kcNC|p|RlC|m
p|kcNC|p|RlCm
p|kcNC|pR|l|C|m
p|kcNC|pR|l|Cm
p|kcNC|pR|lC|m
p|kcNC|pR|lCm
p|kcNC|pRl|C|m
p|kcNC|pRl|Cm
p|kcNC|pRlC|m
p|kcNC|pRlCm
p|kcNCp|R|l|C|m
p|kcNCp|R|l|Cm
p|kcNCp|R|lC|m
p|kcNCp|R|lCm
p|kcNCp|Rl|C|m
p|kcNCp|Rl|Cm
p|kcNCp|RlC|m
p|kcNCp|RlCm
p|kcNCpR|l|C|m
p|kcNCpR|l|Cm
p|kcNCpR|lC|m
p|kcNCpR|lCm
p|kcNCpRl|C|m
p|kcNCpRl|Cm
p|kcNCpRlC|m
p|kcNCpRlCm
pk|c|N|C|p|R|l|C|m
pk|c|N|C|p|R|l|Cm
pk|c|N|C|p|R|lC|m
pk|c|N|C|p|R|lCm
pk|c|N|C|p|Rl|C|m
pk|c|N|C|p|Rl|Cm
pk|c|N|C|p|RlC|m
pk|c|N|C|p|RlCm
pk|c|N|C|pR|l|C|m
pk|c|N|C|pR|l|Cm
pk|c|N|C|pR|lC|m
pk|c|N|C|pR|lCm
pk|c|N|C|pRl|C|m
pk|c|N|C|pRl|Cm
pk|c|N|C|pRlC|m
pk|c|N|C|pRlCm
pk|c|N|Cp|R|l|C|m
pk|c|N|Cp|R|l|Cm
pk|c|N|Cp|R|lC|m
pk|c|N|Cp|R|lCm
pk|c|N|Cp|Rl|C|m
pk|c|N|Cp|Rl|Cm
pk|c|N|Cp|RlC|m
pk|c|N|Cp|RlCm
pk|c|N|CpR|l|C|m
pk|c|N|CpR|l|Cm
pk|c|N|CpR|lC|m
pk|c|N|CpR|lCm
pk|c|N|CpRl|C|m
pk|c|N|CpRl|Cm
pk|c|N|CpRlC|m
pk|c|N|CpRlCm
pk|c|NC|p|R|l|C|m
pk|c|NC|p|R|l|Cm
pk|c|NC|p|R|lC|m
pk|c|NC|p|R|lCm
pk|c|NC|p|Rl|C|m
pk|c|NC|p|Rl|Cm
pk|c|NC|p|RlC|m
pk|c|NC|p|RlCm
pk|c|NC|pR|l|C|m
pk|c|NC|pR|l|Cm
pk|c|NC|pR|lC|m
pk|c|NC|pR|lCm
pk|c|NC|pRl|C|m
pk|c|NC|pRl|Cm
pk|c|NC|pRlC|m
pk|c|NC|pRlCm
pk|c|NCp|R|l|C|m
pk|c|NCp|R|l|Cm
pk|c|NCp|R|lC|m
pk|c|NCp|R|lCm
pk|c|NCp|Rl|C|m
pk|c|NCp|Rl|Cm
pk|c|NCp|RlC|m
pk|c|NCp|RlCm
pk|c|NCpR|l|C|m
pk|c|NCpR|l|Cm
pk|c|NCpR|lC|m
pk|c|NCpR|lCm
pk|c|NCpRl|C|m
pk|c|NCpRl|Cm
pk|c|NCpRlC|m
pk|c|NCpRlCm
pk|cN|C|p|R|l|C|m
pk|cN|C|p|R|l|Cm
pk|cN|C|p|R|lC|m
pk|cN|C|p|R|lCm
pk|cN|C|p|Rl|C|m
pk|cN|C|p|Rl|Cm
pk|cN|C|p|RlC|m
pk|cN|C|p|RlCm
pk|cN|C|pR|l|C|m
pk|cN|C|pR|l|Cm
pk|cN|C|pR|lC|m
pk|cN|C|pR|lCm
pk|cN|C|pRl|C|m
pk|cN|C|pRl|Cm
pk|cN|C|pRlC|m
pk|cN|C|pRlCm
pk|cN|Cp|R|l|C|m
pk|cN|Cp|R|l|Cm
pk|cN|Cp|R|lC|m
pk|cN|Cp|R|lCm
pk|cN|Cp|Rl|C|m
pk|cN|Cp|Rl|Cm
pk|cN|Cp|RlC|m
pk|cN|Cp|RlCm
pk|cN|CpR|l|C|m
pk|cN|CpR|l|Cm
pk|cN|CpR|lC|m
pk|cN|CpR|lCm
pk|cN|CpRl|C|m
pk|cN|CpRl|Cm
pk|cN|CpRlC|m
pk|cN|CpRlCm
pk|cNC|p|R|l|C|m
pk|cNC|p|R|l|Cm
pk|cNC|p|R|lC|m
pk|cNC|p|R|lCm
pk|cNC|p|Rl|C|m
pk|cNC|p|Rl|Cm
pk|cNC|p|RlC|m
pk|cNC|p|RlCm
pk|cNC|pR|l|C|m
pk|cNC|pR|l|Cm
pk|cNC|pR|lC|m
pk|cNC|pR|lCm
pk|cNC|pRl|C|m
pk|cNC|pRl|Cm
pk|cNC|pRlC|m
pk|cNC|pRlCm
pk|cNCp|R|l|C|m
pk|cNCp|R|l|Cm
pk|cNCp|R|lC|m
pk|cNCp|R|lCm
pk|cNCp|Rl|C|m
pk|cNCp|Rl|Cm
pk|cNCp|RlC|m
pk|cNCp|RlCm
pk|cNCpR|l|C|m
pk|cNCpR|l|Cm
pk|cNCpR|lC|m
pk|cNCpR|lCm
pk|cNCpRl|C|m
pk|cNCpRl|Cm
pk|cNCpRlC|m
pk|cNCpRlCm
pkc|N|C|p|R|l|C|m
pkc|N|C|p|R|l|Cm
pkc|N|C|p|R|lC|m
pkc|N|C|p|R|lCm
pkc|N|C|p|Rl|C|m
pkc|N|C|p|Rl|Cm
pkc|N|C|p|RlC|m
pkc|N|C|p|RlCm
pkc|N|C|pR|l|C|m
pkc|N|C|pR|l|Cm
pkc|N|C|pR|lC|m
pkc|N|C|pR|lCm
pkc|N|C|pRl|C|m
pkc|N|C|pRl|Cm
pkc|N|C|pRlC|m
pkc|N|C|pRlCm
pkc|N|Cp|R|l|C|m
pkc|N|Cp|R|l|Cm
pkc|N|Cp|R|lC|m
pkc|N|Cp|R|lCm
pkc|N|Cp|Rl|C|m
pkc|N|Cp|Rl|Cm
pkc|N|Cp|RlC|m
pkc|N|Cp|RlCm
pkc|N|CpR|l|C|m
pkc|N|CpR|l|Cm
pkc|N|CpR|lC|m
pkc|N|CpR|lCm
pkc|N|CpRl|C|m
pkc|N|CpRl|Cm
pkc|N|CpRlC|m
pkc|N|CpRlCm
pkc|NC|p|R|l|C|m
pkc|NC|p|R|l|Cm
pkc|NC|p|R|lC|m
pkc|NC|p|R|lCm
pkc|NC|p|Rl|C|m
pkc|NC|p|Rl|Cm
pkc|NC|p|RlC|m
pkc|NC|p|RlCm
pkc|NC|pR|l|C|m
pkc|NC|pR|l|Cm
pkc|NC|pR|lC|m
pkc|NC|pR|lCm
pkc|NC|pRl|C|m
pkc|NC|pRl|Cm
pkc|NC|pRlC|m
pkc|NC|pRlCm
pkc|NCp|R|l|C|m
pkc|NCp|R|l|Cm
pkc|NCp|R|lC|m
pkc|NCp|R|lCm
pkc|NCp|Rl|C|m
pkc|NCp|Rl|Cm
pkc|NCp|RlC|m
pkc|NCp|RlCm
pkc|NCpR|l|C|m
pkc|NCpR|l|Cm
pkc|NCpR|lC|m
pkc|NCpR|lCm
pkc|NCpRl|C|m
pkc|NCpRl|Cm
pkc|NCpRlC|m
pkc|NCpRlCm
pkcN|C|p|R|l|C|m
pkcN|C|p|R|l|Cm
pkcN|C|p|R|lC|m
pkcN|C|p|R|lCm
pkcN|C|p|Rl|C|m
pkcN|C|p|Rl|Cm
pkcN|C|p|RlC|m
pkcN|C|p|RlCm
pkcN|C|pR|l|C|m
pkcN|C|pR|l|Cm
pkcN|C|pR|lC|m
pkcN|C|pR|lCm
pkcN|C|pRl|C|m
pkcN|C|pRl|Cm
pkcN|C|pRlC|m
pkcN|C|pRlCm
pkcN|Cp|R|l|C|m
pkcN|Cp|R|l|Cm
pkcN|Cp|R|lC|m
pkcN|Cp|R|lCm
pkcN|Cp|Rl|C|m
pkcN|Cp|Rl|Cm
pkcN|Cp|RlC|m
pkcN|Cp|RlCm
pkcN|CpR|l|C|m
pkcN|CpR|l|Cm
pkcN|CpR|lC|m
pkcN|CpR|lCm
pkcN|CpRl|C|m
pkcN|CpRl|Cm
pkcN|CpRlC|m
pkcN|CpRlCm
pkcNC|p|R|l|C|m
pkcNC|p|R|l|Cm
pkcNC|p|R|lC|m
pkcNC|p|R|lCm
pkcNC|p|Rl|C|m
pkcNC|p|Rl|Cm
pkcNC|p|RlC|m
pkcNC|p|RlCm
pkcNC|pR|l|C|m
pkcNC|pR|l|Cm
pkcNC|pR|lC|m
pkcNC|pR|lCm
pkcNC|pRl|C|m
pkcNC|pRl|Cm
pkcNC|pRlC|m
pkcNC|pRlCm
pkcNCp|R|l|C|m
pkcNCp|R|l|Cm
pkcNCp|R|lC|m
pkcNCp|R|lCm
pkcNCp|Rl|C|m
pkcNCp|Rl|Cm
pkcNCp|RlC|m
pkcNCp|RlCm
pkcNCpR|l|C|m
pkcNCpR|l|Cm
pkcNCpR|lC|m
pkcNCpR|lCm
pkcNCpRl|C|m
pkcNCpRl|Cm
pkcNCpRlC|m
pkcNCpRlCm
